afs/afs-ext-base/src/test/java/com/powsybl/afs/ext/base/ImportedCaseTest.java

```diff
@@ -122,4 +143,30 @@ public void test() {
         } catch (Exception ignored) {
         }
     }
+
+    @Test
+    public void testFile() {
+        Folder root = afs.getRootFolder();
+
+        // create project
+        Project project = root.createProject("project");
+        assertNotNull(project);
+
+        // create project folder
+        ProjectFolder folder = project.getRootFolder().createFolder("folder");
+        assertTrue(folder.getChildren().isEmpty());
+
+        ImportedCase importedCase = folder.fileBuilder(ImportedCaseBuilder.class)
+                .withFile(fileSystem.getPath("/work/network.tst"))
+                .withName("test")
+                .build();
+        assertNotNull(importedCase);
+        assertEquals("test", importedCase.getName());
+
+        ImportedCase importedCase2 = folder.fileBuilder(ImportedCaseBuilder.class)
+                .withFile(fileSystem.getPath("/work/network.tst"))
+                .build();
+        assertNotNull(importedCase2);
+        assertEquals("network", importedCase2.getName());
```

Comment:
We keep like this to avoid misunderstanding. These two statements don't give the same result with the new proposal:
```
folder.fileBuilder(ImportedCaseBuilder.class)
    .withName("foo")
    .withFile(fileSystem.getPath("/work/bar.txt"))
    .build();

folder.fileBuilder(ImportedCaseBuilder.class)
     .withFile(fileSystem.getPath("/work/bar.txt"))
     .withName("foo")
     .build();
```